src/ontology/mondo-ingest.Makefile

```diff
@@ -189,8 +189,9 @@ $(TMPDIR)/component-%.json: $(COMPONENTSDIR)/%.owl
 	$(ROBOT) convert -i $< -f json -o $@
 .PRECIOUS: $(TMPDIR)/component-%.json
 
-$(MAPPINGSDIR)/%.sssom.tsv: $(TMPDIR)/component-%.json metadata/mondo.sssom.config.yml
-	sssom parse $< -I obographs-json --prefix-map-mode merged -m metadata/mondo.sssom.config.yml -o $@
+$(MAPPINGSDIR)/%.sssom.tsv:
+	make $(TMPDIR)/component-$*.json metadata/mondo.sssom.config.yml
```

Comment:
```suggestion
	$(MAKE) $(TMPDIR)/component-$*.json metadata/mondo.sssom.config.yml
```
I don't know what the advantage of `$(MAKE)` is. Actually, I don't even see where it's defined. I do see that `MAKE_FAST` is defined in the `Makefile`, though.

Reply:
$(MAKE) uses the same make shell `make` creates a new one, a `make` shell within a `make` shell so to speak. This is a built-in variable

Reply:
Either one is fine here, you can push your commit

Reply:
Ahh, I see, thanks for explaining!
I'll do the commit here just for consistency, then.